### PR TITLE
M2405 Magnum Revolver Rifle

### PR DESCRIPTION
**Pull request by Jaaackx**
## About The Pull Request
Adds the M2405 Magnum Revolver rifle.
.308, 10 rounds, requires both hands, functionally a revolver. Technically a non-canon weapon that was originally designed to be in Fallout Online.
Very lazily balanced as never messed with the current weapon code before, can be tweaked down the road but didn't feel too strong in testing.

dreamseeker_Gi7F42nK3f
qngPzmwHuu

## Pre-Merge Checklist
- [y] You tested this on a local server.
- [y] This code did not runtime during testing.
- [y] You documented all of your changes.

## Changelog
:cl:
add: magnum revolver rifle and a 10 round .308 speedloader
/:cl:

### CODE REVIEW

File: code/modules/projectiles/boxes_magazines/internal/revolver.dm [hunk 1]
```diff
@@ -127,3 +127,10 @@
 	ammo_type = /obj/item/ammo_casing/a45lc
 	caliber = list(CALIBER_45LC)
 	max_ammo = 6
+
+/obj/item/ammo_box/magazine/internal/cylinder/rev308
+	name = ".308 Sporting Revolver Cylinder"
+	ammo_type = /obj/item/ammo_casing/a762/sport
+	caliber = list(CALIBER_762)
+	max_ammo = 10
```
Comment (Superlagg) [inline code/modules/projectiles/boxes_magazines/internal/revolver.dm:135]: I'd prefer this to be something like 8 or 7, that way it doesnt directly compete with the full on semiauto rangemaster.

**Review by Superlagg — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/boxes_magazines/internal/revolver.dm (hunk 1 above)
Comment (Jaaackx) [reply code/modules/projectiles/boxes_magazines/internal/revolver.dm:135]: Rangemaster fires faster, that was the balance I had in mind personally. I can change it but this thing has a mean fire delay currently, it probably doesn't need any nerf.

**Comment by Jaaackx** [comment]
Fixed the compiling issue, forgot a comma, forgot to fix.

**Comment by bearrrrrrrr** [comment]
based based based based based based ba

File: code/modules/projectiles/boxes_magazines/internal/revolver.dm (hunk 1 above)
Comment (Superlagg) [reply code/modules/projectiles/boxes_magazines/internal/revolver.dm:135]: Fair enough!

**Review by Superlagg — APPROVED** [review]

**Pull request merged**